love/src/components/ESS/AuxTel/AuxTelESS.container.jsx

```diff
@@ -99,30 +100,67 @@ const getGroupNames = (salindexList, option) => {
   return [...new Set(subscriptions)];
 };
 
+const prevParseToArraySensors = (prevParse, option) => {
+  const list = Object.values(prevParse[option] ?? {}) ?? [];
+  const objs = Object.values(list).map((values) => {
+    return Object.values(values);
+  });
+
+  const result = Array.prototype.concat.apply([], objs);
+  const sorted = result.sort((a, b) => {
+    return a.sensorName > b.sensorName ? 1 : (
+      a.sensorName < b.sensorName ? -1 : (
+        a.sensorName === b.sensorName && a.indexArr > b.indexArr ? 1 : (
+          a.indexArr < b.indexArr ? -1 : 0
+        )
+      )
+    );
+  });
+  return sorted;
+}
+
+let prevParse = {};
```

Comment:
I think this is not recommended because the variable lives outside the scope of the component and that can potentially lead to state management issues. I think a slightly better way to do this is by setting the:
```javascript
let prevParse = {};
``` 
Inside the `mapStateToProps` function and then pass it as an argument to the `parse` function.